.idea/misc.xml

```diff
@@ -1,4 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CMakeWorkspace" PROJECT_DIR="$PROJECT_DIR$" />
+  <component name="CidrRootsConfiguration">
+    <libraryRoots>
+      <file path="$PROJECT_DIR$/libs" />
+    </libraryRoots>
+    <excludeRoots>
+      <file path="$PROJECT_DIR$/resource" />
+    </excludeRoots>
+  </component>
```

Comment:
Waarom wordt er een idea bestand gewijzigd? Is dit CLion nonsense of is dit met een reden?

Reply:
als je dan linter gebruikt lokaal, probeert hij niet de obj files als code te lezen

Reply:
de andere aanapassingen zijn, alleen opmaak of cast toevoegen, maar kunnen ook gerevert worden, mocht het een probleem zijn.

Reply:
Okay, top